resources/js/components/presenters/party/BuildingsSection.vue

```diff
@@ -0,0 +1,279 @@
+        <autocomplete
+            :label="$t('Add buildings')"
+            id="buildings"
+            :list="buildings.items.map(b => b.id.toString())"
+            @change="toggle"
+            class="mb-8">
+            <template v-for="building in buildings" v-slot:[slugify(building.id)]>
+                <div class="w-full flex items-center justify-between" :key="building.id">
+                    <slot name="label" :item="building.id">
+                        <span>{{ building.id }}</span>
+                    </slot>
+                    <span class="material-icons">
+                        {{ building.isUnlocked() ? 'check_circle_outline' : 'radio_button_unchecked' }}
+                    </span>
+                </div>
+            </template>
+        </autocomplete>
+        <div>
+            <div v-for="building in activeBuildings" :key="building.key()" class="bg-dark-background grid sm:grid-cols-2 md:grid-cols-3 lg:grid-cols-5 gap-px mb-3">
+                <div class="outline-gray row-span-2 sm:col-span-2 lg:col-span-1 flex items-center p-3">
+                    {{ building.id }} {{ $t(building.name) }} {{ $t('Lvl.') }} {{ building.level }}
+                </div>
+                <template v-if="!building.isWrecked()">
+                    <div class="outline-gray flex flex-col items-center p-2 md:row-start-3 lg:row-start-auto" :class="{ 'row-span-2': !$t(building.passive) }">
+                        <add-links-and-icons text="{OPERATIONS}" class="mb-1"/>
+                        <add-links-and-icons class="inline-icons" :text="$t(building.operations) || '-'"/>
+                    </div>
+                    <div class="outline-gray flex flex-col items-center p-2 md:row-start-3 lg:row-start-auto" :class="{ 'row-span-2': !$t(building.passive) }">
+                        <add-links-and-icons text="{DOWNTIME}" class="mb-1"/>
+                        <add-links-and-icons class="inline-icons" :text="$t(building.downtime) || '-'"/>
+                    </div>
+                    <add-links-and-icons v-if="$t(building.passive)" :text="$t(building.passive)" class="inline-icons outline-gray text-center sm:col-span-2 md:row-start-4 lg:row-start-2 lg:col-start-2 p-1"/>
+                </template>
+                <div v-else class="outline-gray flex flex-col items-center p-2 row-span-2 sm:col-span-2 md:row-start-3 lg:row-start-auto">
+                    <add-links-and-icons text="{WRECKED}" class="mb-1"/>
+                    <add-links-and-icons class="inline-icons" :text="$t(building.wrecked) || '-'"/>
+                </div>
+                <div class="outline-gray row-span-2 flex items-center justify-center p-2">
+                    <button v-if="!building.completed" :disabled="building.isWrecked()" @click="upgrade(building)" class="mdc-button mdc-button--raised" :class="{ 'h-auto p-1': !building.lockedUpgrade }">
+                        <add-links-and-icons text="{UPGRADE}"/>
```

Comment:
We should also provide a way to downgrade a building, so users may roll back mistakes.